ErlangInstaller/AppDelegate.swift

```diff
@@ -30,7 +30,7 @@ class AppDelegate: NSObject, NSApplicationDelegate {
             self.mainMenu.scheduleCheckNewReleases()
         }
         
-        let theStoryboard :NSStoryboard? = NSStoryboard(name: "ErlangInstallerPreferences", bundle: nil)
+        let theStoryboard :NSStoryboard? = NSStoryboard(name: NSStoryboard.Name(rawValue: "ErlangInstallerPreferences"), bundle: nil)
```

Comment:
API designers, why use a string when you can enforce a builder :-) Not a criticism of your work, just an observation I saw the exact same pattern in Java Solr API yesterday.

Reply:
NSStoryboard.Name is an alias for a string, I don't think you HAVE to do this.

Reply:
yes! they changed the rules again :man_facepalming: